tests/switchmap_/server/api/routes/test__post.py

```
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import os
+import sys
+import unittest
+from flask import Flask
+from unittest.mock import patch, MagicMock
```

Ruff (0.8.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)